### PR TITLE
sys-fs/duf: new package, add 0.8.1

### PR DESCRIPTION
**Pull request by mrmagic223325**
Bug: https://bugs.gentoo.org/888906
Signed-off-by: Mathis Winterer <mrmagic223325@fedora.email>

### CODE REVIEW

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+DESCRIPTION="Disk Usage/Free Utility - a better 'df' alternative"
+HOMEPAGE="https://github.com/muesli/duf"
+SRC_URI="https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz"
+LICENSE="MIT BSD Apache-2.0"
+SLOT="0"
+IUSE="pie"
+KEYWORDS="~amd64"
+BDEPEND=">=dev-lang/go-1.16"
```
Comment (stha09) [inline sys-fs/duf/duf-0.8.1.ebuild:16]: dev-lang/go dependency is already in go-module eclass. You can remove it here.

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		GOFLAGS+="-buildmode=pie"
+	fi
+	ego build "${GOFLAGS}"
+}
+
+src_install() {
+	dobin duf
+	dodoc README.md
+	doman duf.1
+	default
```
Comment (stha09) [inline sys-fs/duf/duf-0.8.1.ebuild:30]: No need for default src_install.

**Comment by stha09** [comment]
You got me wrong, src_install is needed, just the call to "default" is redundant.

**Comment by mrmagic223325** [comment]
> You got me wrong, src_install is needed, just the call to "default" is redundant.

Bad wording lol.

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="Disk Usage/Free Utility - a better 'df' alternative"
+HOMEPAGE="https://github.com/muesli/duf"
+SRC_URI="https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz"
+LICENSE="MIT BSD Apache-2.0"
```
Comment (ConiKost) [inline sys-fs/duf/duf-0.8.1.ebuild:12]: New line before please. See ```skel.ebuild```

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+SLOT="0"
+IUSE="pie"
+KEYWORDS="~amd64"
+
+src_compile() {
+	GOFLAGS=""
```
Comment (ConiKost) [inline sys-fs/duf/duf-0.8.1.ebuild:18]: ```local``` please

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+inherit go-module
+
+DESCRIPTION="Disk Usage/Free Utility - a better 'df' alternative"
+HOMEPAGE="https://github.com/muesli/duf"
+SRC_URI="https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz"
+LICENSE="MIT BSD Apache-2.0"
+SLOT="0"
+IUSE="pie"
+KEYWORDS="~amd64"
```
Comment (ConiKost) [inline sys-fs/duf/duf-0.8.1.ebuild:15]: KEYWORDS should be before IUSE. See ```skel.ebuild```.

File: sys-fs/duf/duf-0.8.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+DESCRIPTION="Disk Usage/Free Utility - a better 'df' alternative"
+HOMEPAGE="https://github.com/muesli/duf"
+SRC_URI="https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz"
```
Comment (scardracs) [inline sys-fs/duf/duf-0.8.1.ebuild:11]: Isn't
```
SRC_URI="
	https://github.com/muesli/duf/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
	https://github.com/mrmagic223325/duf/releases/download/${PV}/${P}-deps.tar.xz
"
```
enough?

Comment (ConiKost) [reply sys-fs/duf/duf-0.8.1.ebuild]: Yes, it is.

**Comment by ConiKost** [comment]
Please squash into one commit.

**Pull request closed without merging**